src/data/wax_index.rs

```diff
@@ -105,15 +136,20 @@ impl WaxIndex {
 
     /// Package installed, add it to the index.
     ///
-    /// Returns the existing entry as a `package_name@version` String if one exists
+    /// Returns true if an existing entry was updated.
     pub fn insert_entry(
         &mut self,
         entry: String,
         version: Version,
         package_name: String,
-    ) -> Option<String> {
-        self.index
-            .insert(entry, format!("{}@{}", package_name, version))
+    ) -> Option<bool> {
```

Comment:
Can this just return `bool`? I think it will lead to a cleaner API.

Reply:
Discussed offline!  Writing for future reference: I prefer `Option<bool>` to indicate that an underlying failure happened, in this case we can just `unwrap` though because we're getting the time ourselves